www/views/hardware.html

```diff
@@ -1623,35 +1623,56 @@ <h2><span data-i18n="Hardware">Hardware</span>:</h2>
 				</tr>
 			</table>
 		</div>
-		<div id="divphilipshue">
-			<br>
-			<table class="display" id="hardwareparamsphilipshue" border="0" cellpadding="0" cellspacing="20">
+        <div id="divphilipshue">
```

Comment:
Could you undo the formatting changes here? Looks like tabs have been replaced by spaces, not only on this line.
Only changes related to the new hardware please

Also please use tabs instead of spaces

Reply:
The whole file is alternating tab/space. Even the section you here show is originally both types.
![image](https://user-images.githubusercontent.com/3652665/46919791-cdf9e680-cfe4-11e8-8441-cb1f48b66d31.png)

To correct this, the best way would be to do all lines *space -> tab*. Would that be OK?